doc/src/modules/polys/ringseries.rst

```diff
@@ -0,0 +1,87 @@
+.. _polys-ringseries:
+
+=====================================
+Series Manipulation using Polynomials
+=====================================
+
+Any finite Taylor series, for all practical purposes is, in fact a polynomial.
+This module makes use of the efficient representation and operations of sparse
+polynomials for very fast multivariate series manipulations. Typical speedups
+compared to SymPy's ``series`` method are in the range 20-100, with the gap
+widening as the series being handled gets larger. Though the definition of a
+polynomial limits the use of Polynomial module to Taylor series, we extend it
+to allow Laurent and even Puiseux series (with fractional exponents).
+
+All the functions expand any given series on some ring specified by the user.
+Thus, the coefficients of the calculated series depend on the ring being used.
+For example::
+
+    >>> R, x, y = ring('x, y', QQ)
+    >>> rs_sin(x*y, x, 5)
+    -1/6*x**3*y**3 + x*y
+
+``QQ`` stands for the Rational domain. Here all coefficients are rationals.
+Similarly, if a Real domain is used::
+
+    >>> R, x, y = ring('x, y', RR)
+    >>> rs_sin(x*y, x, 5)
+    -0.166666666666667*x**3*y**3 + x*y
+
+All series returned by the functions of this module are objects of the
+``PolyElement`` class. To use them with other SymPy types, convert them  to ``Expr``::
+
+    >>> a = symbols('a')
+    >>> series = rs_exp(x, x, 5)
+    >>> a + series.as_expr()
+    a + x**4/24 + x**3/6 + x**2/2 + x + 1
```

Comment:
Can you also show an example of Laurent and then an example of Puiseux series here as well?